lib/vrl/stdlib/src/parse_host.rs

```diff
@@ -0,0 +1,164 @@
+use std::collections::BTreeMap;
+use tldextract::*;
+use vrl::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct ParseHost;
+
+impl Function for ParseHost {
+    fn identifier(&self) -> &'static str {
+        "parse_host"
+    }
+
+    fn summary(&self) -> &'static str {
+        "parse a host string"
+    }
+
+    fn usage(&self) -> &'static str {
+        indoc! {r#"
+            Parses the provided `value` into TLD, regiestered domain, and subdomains.
+
+            By default included private domains.
```

Comment:
```suggestion
            By default includes private domains.
```

I guess? Not sure, but the current sentence reads weird to me.

Reply:
Agreed, definitely typo